### DISCUSSION TITLE
[Help] 关于 颁发通配符证书 功能的业务流程的理解

### DISCUSSION THREAD

**Discussion by zuilintan** [0xJacky/nginx-ui#574] [Q&A]
Hello，我尝试阅读Doc与源码，大致得到如下流程：

1. 添加 `颁发通配符证书` 后，会创建 `/etc/nginx/ssl` 文件夹，并生成证书
2. `NginxUI `会在初始化时，调用 `InitCronJobs` 方法，启动一个 `GoCron` 自动续期任务（不会在系统中添加任何计划任务，e.g. crontab）
3. 用户在添加 `Website` 时，手动引入 `/etc/nginx/ssl`下的证书

以上，不知道理解是否有误。

从上述流程看，`NginxUI` 自身对 `Nginx` 的配置文件是无侵入的，
如此，哪怕用户同时还在使用 `Certbot`，也可以与 `NginxUI` 并存，按需修改 `Nginx` 配置中的证书路径即可切换，
对吗？

**Comment by 0xJacky** [comment]
是的，不影响的，都是独立的
